examples/blog-tutorial/.eslintrc.js

```diff
@@ -1,4 +1,3 @@
 module.exports = {
-  root: true,
```

Comment:
Because this one was in the example, it was trying to lint `.md` files without having rules/configs/plugins enabled for those kind of files, which caused errors